src/services/transfer/__tests__/transfer-out-util.test.js

```diff
@@ -324,7 +319,7 @@ describe('testTransferOutUtil', () => {
 
     it('should return a fragment unchanged if no other fragment was referenced in it', async () => {
       // given
-      const ehrFragment = JSON.parse(getValidMessageFragment());
+      const ehrFragment = getValidMessageFragment();

```

Comment:
maybe just rename to `fragment` like the rest of the tests.

Reply:
Resolved as part of another commit!